### PR TITLE
Expose count_objects_in_step method

### PR DESCRIPTION
**Pull request by tallenaz**
These will be used in sul-dlss/preservation_robots

### CODE REVIEW

**Comment by tallenaz** [comment]
@ndushay @atz note that I've revised the approach to expose the formerly protected method the convenience methods wrapped.

**Comment by ndushay** [comment]
and yup, I'll approve it this way, too!  heh.

File: spec/workflow_service_spec.rb [hunk 1]
```diff
@@ -464,6 +464,31 @@
...
+      @workflow   = 'sdrIngestWF'
+      @step       = 'start-ingest'
+      @type       = 'waiting'
+      @repository = 'sdr'
+    end
+
+    let(:stubs) do
+      Faraday::Adapter::Test::Stubs.new do |stub|
+        stub.get("/workflow_queue?repository=#{@repository}&workflow=#{@workflow}&#{@type}=#{@step}") do |env|
+          [200, {}, <<-EOXML ]
```
Comment (jmartin-sul) [inline spec/workflow_service_spec.rb]: cool, this just made me learn some things about heredoc!

File: spec/workflow_service_spec.rb [hunk 2]
```diff
@@ -464,6 +464,31 @@
     end
   end
 
+  describe '#count_objects_in_step' do
+    before(:all) do
+      @workflow   = 'sdrIngestWF'
+      @step       = 'start-ingest'
+      @type       = 'waiting'
+      @repository = 'sdr'
```
Comment (atz) [inline spec/workflow_service_spec.rb]: @tallenaz not a big deal, but these read like they should be `let` statements to me

**Pull request merged**